src/dependency-manager/spec/utils/slugs.ts

```diff
@@ -74,7 +74,7 @@ function parseCurry<S extends string, P extends ParsedSlug>() {
 }
 
 export class ComponentSlugUtils extends SlugUtils {
-  public static Description = `${Slugs.ArchitectSlugDescription}; optionally can be prefixed with a valid Architect account and separated by a slash (e.g. architect/component-name).`;
+  public static Description = Slugs.ArchitectSlugDescription;
```

Comment:
I'm worried this will still create ambiguity if we don't remove the component_account_name from ParsedComponentSlug/ParsedComponentVersionSlug. 

Maybe we don't do that as part of this pr. There are enough changes here where I'd like to not break anything and fix the error msgs to not include account.

Reply:
Yeah so that's something that I mentioned in the description above and is kind of complicated - 

> ComponentSlugUtils can drop the component_account_name completely if it's fully deprecated and architect.yml's aren't accepted with an account name anymore, and we can perhaps build an AccountComponentSlug class to handle creating/parsing strings we expect to always contain account_name separate from user component names that don't have account_name.

(I actually meant the `Parsed` version of the slug here)

The reason we can't remove it now is because we use the `ParsedComponentSlug` and `ParsedComponentVersionSlug` to also extract the account name after we call `ComponentSlugUtils.build(this.account, component_name, tag)` in both the CLI and the API. So, the `ComponentSlugUtils` at the moment needs to match `ParsedComponentSlug` with regard to including the account_name at all times.

I think what we may need to do is create another `AccountParsedComponentSlug` we can use to `.build()` slugs with an account name included for backwards compatibility in the API, but also do a bunch of work in the API to separate account name from the component slug itself. I tried to keep this PR pretty focused on just removing all user facing `account-name/component-name` stuff so we can deal with the rest of the mess more cleanly